modules/livox_sdk2/1.2.5.bcr.2/presubmit.yml

```diff
@@ -0,0 +1,19 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - ubuntu2404
+  bazel: [7.x, rolling]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - '--process_headers_in_dependencies'
```

Comment:
This needs `--features=parse_headers` to be effective

Reply:
added https://github.com/bazelbuild/bazel-central-registry/pull/3618/commits/fa09746c89750cf7bbfc1b058d227bf9fcd7c5ab

Reply:
@fmeum add features into BUILD.bazel directly to fix build error https://github.com/bazelbuild/bazel-central-registry/pull/3618/commits/e12cc06035cf9f3d8ade16d86464488a82b27867